shared/common/ui/select/select.module.scss

```diff
@@ -1,14 +1,16 @@
 @import "../../mixins.scss";
 
 .select {
-  --hoverBg: rgba(0, 0, 0, 0.05);
+  --hoverBg: var(--app-bg-gray);
   position: relative;
   display: flex;
   align-items: center;
   border-radius: 4px;
+  color: var(--header-tab-text-colour);
 
   &.fullButton {
     cursor: pointer;
+    padding: 6px 10px;
```

Comment:
This padding only seems to be correct for the header select elements, in other places it breaks the design:
![image](https://user-images.githubusercontent.com/2085287/214571210-df79a116-f3cd-4973-a410-54eec2e7a72e.png)
![image](https://user-images.githubusercontent.com/2085287/214571271-4ea3d402-280f-4c95-9d9f-8cf82742b6b8.png)